### ISSUE TITLE
Building Page: tile perspective quirk

### ISSUE THREAD

**Issue by meetar** [polygon-city/building-database#97]
To reproduce: upload essex.zip

Result:
super-warped perspective on the 3d view
![screen shot 2015-03-12 at 11 41 30 am](https://cloud.githubusercontent.com/assets/459970/6621453/195b9ae4-c8ad-11e4-8f82-c7c074eba1ae.png)

**Comment by robhawkes** [comment]
I don't know what all the fuss is about… it' looks totally normal!

Does this happen with building-based uploads? They're probably using a more correct unit system (metres?) so I'd want to make sure they look ok before I wade in with a workaround. By the sounds of it, it's just to do with the scaling / units and maybe even the initial distance of the camera.

**Comment by robhawkes** [comment]
Ok, so I've looked into this and the only way I could replicate it was by scaling a model to borough / city-wide levels, far larger than any individual building that I know of. We shouldn't really allow this anyway, so while we could fix the camera to support it (the implementation of which is already quite fragile) – buildings that are this big are technically wrong and so I'd suggest we find a way to prevent this even happening during the `/add/location` step (eg. "This building is too large. If this is right, consider splitting it up.").

In the meantime, I've added bc66614 which caps the scaling values so while the rotating plinth isn't fully visible, the camera doesn't get distorted any more.
